sky/jobs/core.py

```diff
@@ -138,6 +140,81 @@ def launch(
                    _disable_controller_check=True)
 
 
+def queue_kubernetes(pod_name: str,
+                     context: Optional[str] = None,
+                     skip_finished: bool = False) -> List[Dict[str, Any]]:
+    """Gets the jobs queue from a specific controller pod.
```

Comment:
This naming is surprising. From the name I thought it's "gets the queue info for an entire k8s cluster". Maybe rename?

Reply:
Updated to `queue_from_kubernetes_pod`